plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -50,136 +32,23 @@
     RAW_SQL_ATTRIBUTE = "raw_sql"
 
 
-@dataclass
-class DbtConfigArgs:
-    """Arguments to load dbt runtime config."""
-
-    project_dir: Optional[str] = None
-    profiles_dir: Optional[str] = None
-    profile: Optional[str] = None
-    target: Optional[str] = None
-    single_threaded: bool = False
-    vars: str = ""
-
-
 class DbtTemplater(JinjaTemplater):
-    """A templater using dbt."""
+    """dbt templater for dbt-osmosis, based on sqlfluff-templater-dbt."""
 
+    # Same templater name as sqlfluff-templater-dbt. It is functionally
+    # equivalent to that templater, but optimized for dbt-osmosis. The two
+    # templaters cannot be installed in the same virtualenv.
```

Comment:
This reads kind of cryptically.

I think saying `dbt templater, leveraging dbt-osmosis` and then:
> From version x.x.x of the dbt templater onwards, we use `dbt-osmosis` to wrap the `dbt` package and reduce complexity here.

...might be clearer. This isn't a different tempalter - it's just the newest version, which offloads a load of complexity.

Might make sense to release this as a 2.0.0 too - given that opportunity presents itself soon?

Reply:
I'll remove this. This code was copied verbatim from a re-implemented dbt templater I wrote that is hosted inside a web service for faster response time. In that case, it was a separate templater also named `dbt`. Not the case here!!